src/main/java/duke/command/AddDeadlineCommand.java

```diff
@@ -0,0 +1,58 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.manager.Storage;
+import duke.manager.TaskList;
+import duke.task.Deadline;
+
+/**
+ * Represents a command that will add a Deadline Task to the TaskList upon execution..
+ */
+public class AddDeadlineCommand extends Command {
```

Comment:
An alternative could be to have all the commands in one class, but I can understand that this might be neater too! :)